### PR TITLE
Removed culture aware parsing of types like Thickness, Matrix, Rect

### PR DESCRIPTION
**Pull request by Gillibald**
Types like Thickness use a comma to seperate values but in some languages like german that can lead to errors.

de-DE("2,2") vs en-US("2.2")

These values dont need to be culture away. We just need a set Format to define them so we should use CultureInfo.InvariantCulture to convert them from string.

### CODE REVIEW

**Review by grokys — APPROVED** [review]
This looks like the correct thing to do to me. I'd also like @aelij who wrote the `StringTokenizer` to take a look at it and see if he agrees.

**Comment by grokys** [comment]
Good point @aelij. And now you mention it, I've just noticed that `StringTokenizer` only separates on `,`. If we're going to follow WPF etc then `"10 0 0 0"` should also be a valid margin. So I think `StringTokenizer` should split on `,` and also whitespace.

**Comment by grokys** [comment]
Oh ok, ignore me then :)

**Comment by aelij** [comment]
@Gillibald `IsWhitespace` is not magic, it's just `char.IsWhitespace` :) I would not remove it since it takes care of leading/trailing whitespace. Also, there's currently no need for a list of separators.

Also I think we should complete this PR and if necessary make changes to the tokenizer (including those I suggested) in a different PR.

**Comment by Gillibald** [comment]
We can make changes to StringTokenizer in a separate PR that's fine. Just in case you want to parse FontFamily (A A; B B) containing whitespaces should not be removed. Just want to have control over whether we want separate by whitespace or not.

**Comment by grokys** [comment]
So @aelij, do you give the 👍 to this PR as-is? Or do you want changes?

**Comment by aelij** [comment]
@Gillibald I'm not aware of such a use case, and I wouldn't make changes just for the sake of changes...

File: src/Avalonia.Visuals/CornerRadius.cs [hunk 1]
```diff
@@ -53,39 +54,34 @@ public override string ToString()
...
 
-        public static CornerRadius Parse(string s, CultureInfo culture)
+        public static CornerRadius Parse(string s)
         {
-            var parts = s.Split(new[] { ',', ' ' }, StringSplitOptions.RemoveEmptyEntries)
-                .Select(x => x.Trim())
-                .ToList();
-
-            switch (parts.Count)
+            using (var tokenizer = new StringTokenizer(s, CultureInfo.InvariantCulture, exceptionMessage: "Invalid Thickness"))
```
Comment (amis92) [inline src/Avalonia.Visuals/CornerRadius.cs]: @Gillibald the `exceptionMessage` doesn't seem correct?

**Pull request merged**